mizar/dp/mizar/workflows/builtins/pods/create.py

```diff
@@ -20,6 +20,7 @@
 # THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 import logging
+import json
```

Comment:
Sorry for a very basic question: why do we need to change pod workflows here?  My understanding is for the 9/15 release Enju/Hong's network controller will call endpoint operator directly to create endpoint objects. In that case we don't need pods workflow anymore, right?

Reply:
Get the answer from scrum meeting. It's for native kubernetes.  We will discuss the design after current release. Close the comment.